README.md

```diff
@@ -125,5 +164,9 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+A schema is the outline for the table.  It states what will go in each column and
+whether they are null, integers or characters. A one to many relationship is represented
```

Comment:
Null does not reference the data type stored in the column. You're probably referring to `NOT NULL`, which is a constraint we place on a column to indicate that it should never be empty.